nilearn/maskers/tests/test_multi_nifti_masker.py

```diff
@@ -193,25 +193,6 @@ def test_mask_strategy_errors():
         mask.fit(imgs)
 
 
-@pytest.mark.parametrize('strategy',
-                         [f'{p}-template' for p in
-                          ['whole-brain', 'gm', 'wm']])
-def test_compute_multi_gray_matter_mask(strategy):
```

Comment:
I might be missing something, but I think this test isn't calling the deprecated function :  

https://github.com/nilearn/nilearn/blob/e19b37dcb74cc3819b5b60c49f0917f958c7aacb/nilearn/maskers/multi_nifti_masker.py#L26-L33

Reply:
Yes you are completely right. I think then it just needs a name change. Would you agree?

Reply:
> Yes you are completely right. I think then it just needs a name change. Would you agree?

Yup ! The name is misleading !